### ISSUE TITLE
[OAKTrust] Communities and Collections Page

### ISSUE THREAD

**Issue by devangm** [TAMULib/dspace-angular#203]
**URL:** https://oaktrust-dev.library.tamu.edu/community-list
**WCAG 2.1 AA compliant? :** No
**Violation:**
**Issues:** NVDA navigation
**Notes:** navigation is either very hard or impossible with this tree structure. Can't select an element after "colleges and schools" is opened.

**Comment by tamu-sad-iii** [comment]
A race condition between the cdk-tree with the default behavior by selecting the parent element due to cdk-tree-node select programmatically changing the tabindex of the node element.

The requirement to satisfy this accessibility would be that it does not select the parent element when triggering the expand collapse method handler.

**Comment by tamu-sad-iii** [comment]
From here create new issue to resolve the race condition and approving this post demos testing.
